tests/http/cache-control.yml

```diff
@@ -0,0 +1,44 @@
+config: !file tests/http/config/simple.graphql
+name: Cache Control Header Test
+mock:
+  - request:
+      url: http://jsonplaceholder.typicode.com/users/1
+    response:
+      body:
+        id: 1
+        name: foo
+      headers:
+        Cache-Control: "max-age=3600, public"
+  - request:
+      url: http://jsonplaceholder.typicode.com/users/2
+    response:
+      body:
+        id: 2
+        name: bar
+      headers:
+        Cache-Control: "max-age=7200, public"
+assert:
+  - request:
+      method: POST
+      url: http://localhost:8080/graphql
+      body:
+        query: query { user { name } }
+    response:
+      body:
+        data:
+          user:
+            name: foo
+      headers:
+        Cache-Control: "max-age=3600, public"
+  - request:
+      method: POST
+      url: http://localhost:8080/graphql
+      body:
+        query: query { user { name } }
```

Comment:
The query should data from two different API calls and minimum of the `cache-control` headers should received at downstream.

Reply:
The goal is to test if the headers are calculated properly

Reply:
@amitksingh1490
mapping values are not allowed. I don't know how to use two difference API calls here, it would be really helpful if you could assist me!
`query: query { user(id: "1") { name } }`
`query: query { user(id: "2") { name } }`

Reply:
```
query: query { 
 u1: user(id: "1") { name }
 u2: user(id: "1") { name }
}
```

Reply:
> ```graphql
> query: query { 
>  u1: user(id: "1") { name }
>  u2: user(id: "1") { name }
> }
> ```

Still the test is getting failed when I use this query on both the requests in assert

I have also tried

> ```graphql
> query: query { 
>  u1: user(id: "1") { name }
> }
> ```

> ```graphql
> query: query { user(id: "1") { name } }
> ```

Caused by: mapping values are not allowed in this context